handler.go

```diff
@@ -329,10 +328,8 @@ func (h FilteredHttpRequestHandler) Handle(req *http.Request, metrics Metrics) (
 		_, ok := h.allowedOrigins[req.Host]
 		if !ok {
 			metrics.Fire(metricsResultTargetRequestForbidden)
-			if h.logForbiddenErrors {
-				// to allow clients to fix improper third party urls usage (e.g. to change URLs from our direct s3 refs to CDN)
-				log.Printf("TargetForbiddenError: %s, %s", req.Host, req.URL)
-			}
+			// to allow clients to fix improper third party urls usage (e.g. to change URLs from our direct s3 refs to CDN)
+			slog.Debug("TargetForbiddenError", "host", req.Host, "URL", req.URL)
```

Comment:
My instinct is that this should be an Error(), not Debug(). It sounds like your interpretation of `logForbiddenErrors` is to make the logs noisier and that suggests Debug()? That also makes sense i think.

Reply:
The existing behavior was to allow operators to opt into extra diagnostic logs by setting a variable. Opting into `LOG_LEVEL=debug` instead of `VERBOSE=true` seems consistent with that.

Beyond that, this should not be logged at error level, because IIUC this case represents a _client_ error, not a server one. You wouldn't want to let a client mistake generate lots of errors in your own logs or metrics.